operations/security_and_authentication/Keycloak_User_Management_with_Kcadm.md

```diff
@@ -0,0 +1,305 @@
+# Keycloak User Management with kcadm.sh
+
```

Comment:
General notes:
- All command prompts in the code blocks should be ncn-m001
- Only one space is needed at the beginning of each sentence
- Add a blank line between lines of text and code blocks